.travis.yml

```diff
@@ -1,29 +1,71 @@
 language: cpp
-os:
-- linux
-- osx
-compiler:
-- clang
-- gcc
+
+matrix:
+  include:
+     # Coverage
+     - os: osx
```

Comment:
Yes, the clang++ on OS X works but @flippmoke found that the clang on travis linux boxes did not play well with coverage (likely some missing libraries). So, running on OS X is my recommendation for now (even though its a bit slow).